src/components/pages/AdminAddCourse/AdminAddCoursesForm.js

```diff
@@ -9,22 +13,40 @@ import {
   Checkbox,
   InputNumber,
   DatePicker,
+  Button,
   Modal,
   TimePicker,
   Select,
 } from 'antd';
 import TextArea from 'antd/lib/input/TextArea';
 const { Option } = Select;
 
-const daysOfWeek = [
-  'Monday',
-  'Tuesday',
-  'Wednesday',
-  'Thursday',
-  'Friday',
-  'Saturday',
-  'Sunday',
-];
+const initialFormValues = {
+  course_id: '',
+  course_name: '',
+  course_description: '',
+  days_of_week: [],
+  max_size: '',
+  max_age: '',
+  min_age: '',
+  start_date: '',
+  end_date: '',
+  start_time: '',
+  end_time: '',
+  number_of_sessions: '',
+  instructor_id: '',
+};
+
+// may be necessary for auto-pop
```

Comment:
This array was originally being used to create the check boxes in the form. It looks like that has been done below now, so this array can be removed.